industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/irak4_s3/protein.pdb

```diff
@@ -0,0 +1,4707 @@
+ATOM   3159  HG3 PRO A 366      52.826  22.464 -12.912  1.00 62.67           H  
+ATOM   3160  HG2 PRO A 366      52.712  23.471 -11.467  1.00 62.67           H  
+ATOM   3161  HD2 PRO A 366      51.836  24.961 -13.083  1.00 58.22           H  
+ATOM   3162  HD3 PRO A 366      51.021  23.673 -13.969  1.00 58.22           H  
+ATOM   3163  N   LYS A 367      50.565  24.617  -9.457  1.00 56.58           N  
+ATOM   3164  CA  LYS A 367      50.625  25.258  -8.142  1.00 55.24           C  
+ATOM   3165  C   LYS A 367      49.234  25.551  -7.541  1.00 55.25           C  
+ATOM   3166  O   LYS A 367      49.129  25.633  -6.318  1.00 55.74           O  
+ATOM   3167  CB  LYS A 367      51.478  26.542  -8.256  1.00 58.03           C  
+ATOM   3168  CG  LYS A 367      52.945  26.259  -8.642  1.00 69.05           C  
+ATOM   3169  CD  LYS A 367      53.713  27.491  -9.145  1.00 76.40           C  
+ATOM   3170  CE  LYS A 367      55.157  27.140  -9.534  1.00 79.25           C  
+ATOM   3171  NZ  LYS A 367      55.849  28.279 -10.160  1.00 91.88           N1+
+ATOM   3172  H   LYS A 367      50.919  25.174 -10.223  1.00 56.58           H  
+ATOM   3173  HA  LYS A 367      51.137  24.573  -7.465  1.00 55.24           H  
+ATOM   3174  HB3 LYS A 367      51.468  27.084  -7.309  1.00 58.03           H  
+ATOM   3175  HB2 LYS A 367      51.022  27.209  -8.987  1.00 58.03           H  
+ATOM   3176  HG3 LYS A 367      52.993  25.493  -9.416  1.00 69.05           H  
+ATOM   3177  HG2 LYS A 367      53.461  25.839  -7.777  1.00 69.05           H  
+ATOM   3178  HD3 LYS A 367      53.717  28.265  -8.380  1.00 76.40           H  
+ATOM   3179  HD2 LYS A 367      53.196  27.919 -10.004  1.00 76.40           H  
+ATOM   3180  HE3 LYS A 367      55.166  26.311 -10.242  1.00 79.25           H  
+ATOM   3181  HE2 LYS A 367      55.718  26.820  -8.655  1.00 79.25           H  
+ATOM   3182  HZ1 LYS A 367      56.792  28.008 -10.398  1.00 91.88           H  
+ATOM   3183  HZ2 LYS A 367      55.360  28.555 -10.999  1.00 91.88           H  
+ATOM   3184  HZ3 LYS A 367      55.881  29.060  -9.515  1.00 91.88           H  
+ATOM   3185  N   SER A 368      48.195  25.672  -8.389  1.00 48.92           N  
+ATOM   3186  CA  SER A 368      46.799  25.852  -7.982  1.00 47.97           C  
+ATOM   3187  C   SER A 368      46.211  24.590  -7.324  1.00 49.32           C  
+ATOM   3188  O   SER A 368      45.492  24.719  -6.334  1.00 47.58           O  
+ATOM   3189  CB  SER A 368      45.942  26.240  -9.199  1.00 49.96           C  
+ATOM   3190  OG  SER A 368      46.394  27.442  -9.787  1.00 59.97           O  
+ATOM   3191  H   SER A 368      48.353  25.601  -9.385  1.00 48.92           H  
+ATOM   3192  HA  SER A 368      46.760  26.664  -7.255  1.00 47.97           H  
+ATOM   3193  HB3 SER A 368      44.903  26.381  -8.903  1.00 49.96           H  
+ATOM   3194  HB2 SER A 368      45.952  25.451  -9.953  1.00 49.96           H  
+ATOM   3195  HG  SER A 368      46.55
```

Comment:
Note for further investigation - this seems like an NME but named as an ALA? HB comments it could be due to the original Schrodinger inputs being incorrectly named.

Reply:
I checked - this is true for both both irak4_s2 and irak4_s3, the schrodinger inputs just have the wrong residue name. In this case I would ask if you can rename the residues to NME please?

Reply:
Renamed Ala to NME and changed atom names